src/ansys/geometry/core/designer/design.py

```diff
@@ -1220,3 +1220,139 @@ def _update_design_inplace(self) -> None:
 
         # Read the existing design
         self.__read_existing_design()
+
+    def update_from_tracker(self, tracker_response):
+        """
+        Update the design with the changed bodies while preserving unchanged ones.
+
+        Parameters
+        ----------
+        changed_bodies : list[dict]
+            A list of dictionaries representing changed body information. Each dictionary
+            should contain 'id', 'name', and 'is_surface' keys.
+        """
```

Comment:
I have the feeling that the parameter name changed to "tracker_response" instead of "changed_bodies"